models/_sector/nft/trades/ethereum/platforms/archipelago_ethereum_base_trades.sql

```diff
@@ -0,0 +1,78 @@
+{{ config(
+    schema = 'archipelago_ethereum',
+    alias ='base_trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_number','tx_hash','sub_tx_trade_id'],
+    )
+}}
+{% set project_start_date='2022-6-20' %}
+
+WITH
+trade_events as (
+    SELECT * FROM {{ source('archipelago_ethereum','ArchipelagoMarket_evt_Trade') }}
+    {% if is_incremental() %}
+    WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    WHERE evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+),
+token_events as (
+    SELECT * FROM {{ source('archipelago_ethereum','ArchipelagoMarket_evt_TokenTrade') }}
+    {% if is_incremental() %}
+    WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    WHERE evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+),
+fee_events as (
+    SELECT
+        evt_block_number
+        , tradeId
+        , sum(amount) filter (
+            where recipient not in ('0xa76456bb6abc50fb38e17c042026bc27a95c3314','0x1fc12c9f68a6b0633ba5897a40a8e61ed9274dc9')
+            ) as royalty_amount
+        , sum(amount) filter (
+            where recipient in ('0xa76456bb6abc50fb38e17c042026bc27a95c3314','0x1fc12c9f68a6b0633ba5897a40a8e61ed9274dc9')
+            ) as platform_amount
+    FROM {{ source('archipelago_ethereum','ArchipelagoMarket_evt_RoyaltyPayment') }}
+    {% if is_incremental() %}
+    WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    WHERE evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+    GROUP BY evt_block_number, tradeId
+)
+
+SELECT
+    date_trunc('day',trade.evt_block_time) as block_date
+    ,trade.evt_block_number as block_number
+    ,trade.evt_tx_hash as tx_hash
+    ,trade.contract_address as project_contract_address
+    ,CAST(null as varchar(5)) as trade_category
+    ,'secondary' as trade_type
+    ,trade.buyer
+    ,trade.seller
+    ,tok.tokenAddress as nft_contract_address
+    ,tok.tokenId as nft_token_id
+    ,1 as nft_amount
```

Comment:
apply accepted value(s) test in schema file? i believe markdown file said only `1` across the board?